docs/user/reference/resource-compute/list-of-commands-available-on-a-compute-resource-provisioned-by-juju/list-of-juju-introspect-macros/juju_agent.md

```diff
@@ -1,4 +1,42 @@
 (juju_agent)=
 # `juju_agent`
 
-TBA
+```{toctree}
+:hidden:
+
+list-of-juju-introspect-macros/index
+
+```
```

Comment:
The TOC tree isn't necessary. Compare, for example, to the source for `juju_engine_report`: https://canonical-juju.readthedocs-hosted.com/en/latest/_sources/user/reference/resource-compute/list-of-commands-available-on-a-compute-resource-provisioned-by-juju/list-of-juju-introspect-macros/juju_engine_report.md.txt

Reply:
Thanks, removed.